src/utils/venue.ts

```diff
@@ -62,10 +62,6 @@ export const createJazzbar = (values: FormValues): JazzbarVenue => {
   return {
     template: VenueTemplate.jazzbar,
     name: values.name || "Your Jazz Bar",
-    mapIconImageUrl: urlFromImage(
-      "/default-profile-pic.png",
-      values.mapIconImageFile
-    ),
```

Comment:
We've removed this code from here, but it looks like `mapIconImageFile` is still going to exist in `FormValues` (and likely related code), which we presumably should also be cleaning up here given it seems to be so tightly related?